05 - coder-files/src/components/Button.jsx

```diff
@@ -0,0 +1,12 @@
+export default function Button({ position, children, onClick }) {
+  const handleClick = () => onClick(position);
+
+  return (
+    <button
+      onClick={handleClick}
+      className="bg-[#FEC800] px-5 py-2 rounded-sm sm:text-xl"
+    >
+      {children}
```

Comment:
why passing children in button, does the button receive any HTML node or elements?